src/Utils.cpp

```diff
@@ -103,13 +106,7 @@ bool utils::IsHeader(const std::string_view& path)
 
 std::string utils::GetNicePath(const std::string_view& path)
 {
-    std::string res(path);
-    ForwardSlashify(res);
-    if (BeginsWith(res, s_CurrentDir))
-        res = res.substr(s_CurrentDir.size(), res.size() - s_CurrentDir.size());
```

Comment:
For me the local paths were already relative.
But this code also fails to fulfill its purpose if the workdir doesn't happen to be the build dir.